src/tslint-cli.ts

```diff
@@ -254,11 +254,11 @@ const processFiles = (files: string[], program?: ts.Program) => {
 
     const lintResult = linter.getResult();
 
-    if (lintResult.failureCount > 0) {
-        outputStream.write(lintResult.output, () => {
+    outputStream.write(lintResult.output, () => {
+        if (lintResult.failureCount > 0) {
             process.exit(argv.force ? 0 : 2);
-        });
-    }
+        }
+    });
```

Comment:
This has the surprising side effect that a lint with nothing to report now prints a `0x0a` to stdout, when it used to have no output.